v2/book/examples/AssemblyMetadata.Generators.html

```
@@ -0,0 +1,53 @@
+
+<h1>RSCG nr 270 :  AssemblyMetadata.Generators</h1>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)